spec/controllers/admin/bulk_line_items_controller_spec.rb

```diff
@@ -59,15 +59,13 @@
         end
 
         it "formats final_weight_volume as a float" do
-          expect(json_response['line_items'].map{ |line_item|
-                   line_item['final_weight_volume']
-                 }.all?{ |fwv| fwv.is_a?(Float) }).to eq(true)
+          expect(json_response['line_items'].pluck(:final_weight_volume).all?{ |fwv|
+                   fwv.is_a?(Float)
+                 }).to eq(true)
         end
 
         it "returns distributor object with id key" do
-          expect(json_response['line_items'].map{ |line_item|
-                   line_item['supplier']
-                 }.all?{ |d| d.key?('id') }).to eq(true)
+          expect(json_response['line_items'].pluck(:supplier).all?{ |d| d.key?('id') }).to eq(true)
         end
       end
```

Comment:
I reckon these expectiations could be refactored even more..

Reply:
It's been addressed here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/12599/commits/ff10a36e4a4b2077c9549511c25e9fe1903f8d09

Reply:
It's been fixed here 🙂: https://github.com/openfoodfoundation/openfoodnetwork/pull/12599/commits/ff10a36e4a4b2077c9549511c25e9fe1903f8d09